frontend/src/screens/recordingModule.js

```diff
@@ -52,16 +49,29 @@ class RecordingModule extends Component {
   };
 
   _StopRecord = async () => {
-    this.setState({ 
-      recording: false,
-      playback: true
-    }, () => {
-      this.cam.stopRecording();
-    });
+    if (this.state.time < 10) {
+      Alert.alert(
+        "Warning",
+        "Please record more than 10 seconds.",
+        [{
+          text: 'OK', onPress: () => {
+            this.setState({
+              video: null,
+              recording: false,
+              time: 0
```

Comment:
the stopTimer function sets time to 0 already so u dont need this line

Reply:
True, you right